### PR TITLE
Run acceptance tests in travis

### PR DESCRIPTION
**Pull request by tvpartytonight**
This PR adds configuration to the travis yml to run `rake acceptance`.

### CODE REVIEW

**Comment by petems** [comment]
I think we'll have to do something to make this work, because I think the Vox settings will just overwrite this when modulesync happens...

@bastelfreak am I right?

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +23,9 @@ matrix:
   - rvm: 2.4.4
     bundler_args: --without system_tests development release
     env: PUPPET_VERSION="~> 5.0" CHECK=build DEPLOY_TO_FORGE=yes
+  - rvm: 2.5.1
+    bundler_args: --without development release
+    env: CHECK=acceptance
```
Comment (pcarlisle) [inline .travis.yml]: all the changes in this file need to go in the sync yaml

Comment (tvpartytonight) [reply .travis.yml]: I manually just synced the added cell without using `pdk update`...is that ok?

**Comment by tvpartytonight** [comment]
@petems I don't think there are any problems with modulesync; there's an obtuse error when mocking out the `http_instance` method call that I fixed, so it passes travis now.

**Comment by petems** [comment]
@tvpartytonight Sorry should have clarified, not a test error, but the The VoxPop modulesync will overwrite the Travis config AFAIK (https://github.com/voxpupuli/modulesync_config) 

But I'm super behind the times so might not be the case anymore 🤷‍♂️

**Comment by Magisus** [comment]
It looks like maybe the Travis file is generated from this ERB template: https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.travis.yml.erb. So I wonder if it would overwrite the `services: docker` field we added. Might we need to update that template to allow for specifying things like docker?

File: spec/acceptance/nodesets/docker/CertsDockerfile [hunk 2]
```diff
@@ -0,0 +1,67 @@
+FROM alpine:3.8
```
Comment (bastelfreak) [inline spec/acceptance/nodesets/docker/CertsDockerfile]: We normally try to generate the dokerfiles on the fly by using https://github.com/puppetlabs/beaker-hostgenerator, but this one looks pretty specific, so a custom file is fine.

File: .travis.yml (hunk 1 above)
Comment (bastelfreak) [reply .travis.yml]: We don't yet use pdk, we currently use https://github.com/voxpupuli/modulesync_config/ as template for our modules.

**Comment by bastelfreak** [comment]
Hey people, thanks for enabling the acceptance tests. I will merge this now and afterwards have a look at the modulesync integration.

**Pull request merged**